java-management/jms-kubernetes-support/jms-kubernetes-support.md

```diff
@@ -0,0 +1,58 @@
+# JMS Kubernetes Support 
+
+## Introduction
+
+This guide will walk you through setting up and using Oracle Java Management Service (JMS) to monitor and manage Java applications running in Kubernetes clusters. JMS Kubernetes support extends your existing fleet management capabilities to containerized environments.
+
+**Estimated Time:** 35 minutes
+
+### Objectives
+
+In this guide, you will:
+
+* Set up JMS monitoring for Kubernetes clusters
+* Monitor Java applications running in containers
+* Manage your Kubernetes-based Java workloads through JMS
+
+### Prerequisites
+
+* Oracle Cloud Infrastructure account with appropriate permissions
+* Access to a Kubernetes cluster 
+* Java applications running in containers with JDK 
+* Basic familiarity with Kubernetes concepts
+
+## Task 1: Download the Kubernetes Agent
+
+1. Open the navigation menu, navigate to **Observability & Management**. Under **Java Management**, click **Fleets**. Select the fleet that you have configured in [Lab 3](?lab=setup-a-fleet)
+
+  ![Console navigation to Java Management Service](images/console-navigation-jms.png)
+
+2. In your fleet details page, click **Download Agent Installer**.
+
+   ![Download Agent Installer](images/download-agent-installer.png)
+
+3. Select **Kubernetes Agent Installation**, and follow the steps provided in the panel.
+
+   ![Kubernetes Agent Installation](images/kubernetes-agent-installation.png)
+
+4. Under **Managed Instances**,once the installation is successfully completed, an entry for the newly added agent should appear.
+
+   ![Kubernetes Agent Installation Verification](images/kubernetes-agent-installation-verification.png)
```

Comment:
Under Lab 6 Install Management Agent on your Managed Instances, there is a Task 4 Verify JMS Plugin Deployment, Task 5 Verify the successful association of Managed Instance to your fleet, Task 6 Verify detection of Java applications and runtimes. 

Suggestion to show the user how to verify that agent installation is successful and how to verify JMS plugin is deployed. 

Suggestion to show the user how to set up Java containers to run and how to verify that they are running. Would be nice to close that loop. 

Clearer to split out into different Tasks under this Lab for each step.